### PR TITLE
minify HTML docs, CSS/JS assets; crush image assets (on deploy) (fixes #44)

### LINKED ISSUE #44: Minify assets (CSS, JS)

### PR DESCRIPTION
**Pull request by cvan**

### CODE REVIEW

**Comment by cvan** [comment]
I need to update this so the minification and compression happens only upon *deploy*, not when *starting* the server every time.

**Comment by cvan** [comment]
note that this doesn't concatenate the JS bundles, but it does minify the JS + CSS and even the HTML, etc. I don't want the JS scripts to get concatenated yet as single bundles. want to keep the JS minimal here, so I won't be fiddling too much with it.

File: _config.prod.yml [hunk 1]
```diff
@@ -149,3 +149,28 @@ aframe_version: 0.2.0
...
+  exclude:
+    - '*.min.css'
+
+js_minifier:
+  enable: true
+  mangle: true
+  output:
+  compress:
+  exclude:
+    - '*.min.js'
```
Comment (cvan) [inline _config.prod.yml]: we should exclude the `releases/` dir

**Comment by ngokevin** [comment]
I hadn't seen this! But I pretty much did the same thing.

To do it only on deploy, we have your local config dist file we can disable from.

**Pull request closed without merging**